sympy/polys/polymatrix.py

```diff
@@ -281,7 +281,7 @@ def nullspace(self):
             raise ValueError("PolyMatrix nullspace is only for ground field elements")
         dm = self._dm
         K, Kx = self.domain, self.ring
-        dm_null_rows = dm.convert_to(K).nullspace().convert_to(Kx)
+        dm_null_rows = dm.convert_to(K).nullspace(normalize=True).convert_to(Kx)
```

Comment:
This is also just preserving existing behaviour inside Risch. I don't know if Risch can use unnormalised nullspace. If so it would definitely be better to do so.

I also realised while looking at this that in Risch there are many very sparse matrices of only rational numbers. The `PolyMatrix` class that is used to represent those internally uses a dense DomainMatrix but the linear algebra parts of Risch could be a lot faster if it used a sparse DomainMatrix.